src/gossip/service.zig

```diff
@@ -125,16 +129,14 @@ pub const GossipService = struct {
 
     const Entrypoint = struct { addr: SocketAddr, info: ?ContactInfo = null };
 
-    const Self = @This();
```

Comment:
we actually want to keep everything as `Self` throughout the codebase for consistency & style sake

Reply:
See my reply to the comment on `RunHandles`.